### ISSUE TITLE
Thumbnail as ablum art is not working

### ISSUE THREAD

**Issue by NiklasEsser** [moshfeu/y2mp3#75]
**Describe the bug**
The preference 'Set the video's thumnail as album art (for mp3 only)' doesn't work correctly.

**To Reproduce**
1. Go to 'https://github.com/moshfeu/y2mp3/releases/tag/v2.5.4'
2. Click on 'y2mp3.Setup.2.5.4.exe'
3. Start the download
4. Execute/Double-Click the program 'y2mp3.Setup.2.5.4.exe'
5. Wait until the installation has completed
6. Start 'y2mp3.exe' (only if not automatically after step 5)
7. (Install FFMpeg if not already done)
8. Set the preferences like this (expect 'Downloads folder')
![Preferences](https://user-images.githubusercontent.com/40426398/116139315-2d68ff00-a6d6-11eb-9220-e999c3ee154f.jpg)

9. Copy this example Youtube URL 'https://www.youtube.com/watch?v=WyTFU-BHO2g'
10. Youtube URL should be auto pasted
11. Start the search
12. Click on 'DOWNLOAD' or 'DOWNLOAD ALL (1)'
13. Go to your 'Downloads folder' path
14. Open the example mp3 file 'Nightcore - Ocean Eyes - (Lyrics).mp3' with an application that the album art shows (my example is the 'Windows Media Player')
11. See error

**Expected behavior**
This black square as ablum art.
![NewBug2](https://user-images.githubusercontent.com/40426398/116141214-acf7cd80-a6d8-11eb-8420-a12c842a6a35.jpg)

**Comment by moshfeu** [comment]
I'll look on it asap.

**Comment by moshfeu** [comment]
Not asap after all, sorry..
If I'll have time in the future, I might get into it
Meantime if someone wants to fix it, I'll be more than happy
